src/python/turicreate/data_structures/sframe.py

```diff
@@ -1270,6 +1270,9 @@ def read_csv(cls,
             continue parsing skipping lines which fail to parse correctly.
             A sample of the first 10 encountered bad lines will be printed.
 
+        nrows_to_infer: integer
```

Comment:
Let's have the ordering of parameters in the docstring match the order of parameters in the function definition.